src/dlls/mscoree/coreclr/README.md

```diff
@@ -0,0 +1,7 @@
+dump\_helper\_resource.bin in this folder is a text file with a single 0 byte appended on the end using a hex-editor. It is unlikely it will need to be modified frequently if at all,
+but if that changes we can always create a little nicer tooling for it. 
+
+dump\_helper\_resource.bin is used to populate the DUMP\_HELPER resource inside coreclr.dll on Windows. When an application crashes, Windows MinidumpWriteDump is planning to scan
```

Comment:
The text indicates that "Windows MinidumpWriteDump is planning to scan". Does Windows currently support this change? And if so, which versions (Windows 7, 8, 10?) Or is this support coming in a future Windows update? And if so, will it also include Windows 7 and 8?